### DISCUSSION TITLE
Distributing log sources into multiples Logstashes

### DISCUSSION THREAD

**Discussion by esbat-max** [Security-Onion-Solutions/securityonion#5727] [Unsupported Versions]
Dear all,

I am using security onion 2, when I add a log source to SO by default it is taking into *.SO.* the default logstash.
My question is how to distribute the log sources among multiple logstashes that are builtin in Security onion.

Thank you,

**Comment by n0th1nt0lOOz** [comment]
What is the mode you choose? Standalone or Heavy?

**Comment by esbat-max** [comment]
Hello,

I am using so in standalone mode.

Thanks,

Sent from my iPhone

On Sep 30, 2021, at 9:27 PM, tientmse62290 ***@***.******@***.***>> wrote:


What is the mode you choose? Standalone or Heavy?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Security-Onion-Solutions/securityonion/discussions/5727#discussioncomment-1410630>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/APZ42V5UN7A3BWWBI7R6K43UEST2TANCNFSM5FCR5PKQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by n0th1nt0lOOz** [reply]
@esbat-max in standalone mode there is only one logstash node. You could find the architecture and Logs in document

**Comment by weslambert** [comment]
When you say distribute into multiple Logstash(es)..do you mean:

- Divert the log source into a differently named index (other than the default `so-`-prefixed index name
- Divert the log source into a separate Logstash pipeline (for sending somewhere else or locally?)
- Both
- Neither

**Comment by esbat-max** [comment]
Hello,

Thanks for your reply.

Exactly, i mean indexing other than the default SO.

br,
Elie

Sent from my iPhone

On Oct 5, 2021, at 2:48 PM, weslambert ***@***.******@***.***>> wrote:


When you say distribute into multiple Logstash(es)..do you mean:

  *   Divert the log source into a differently named index (other than the default so--prefixed index name
  *   Divert the log source into a separate Logstash pipeline (for sending somewhere else or locally?)
  *   Both
  *   Neither

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Security-Onion-Solutions/securityonion/discussions/5727#discussioncomment-1429704>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/APZ42VZNAAWIPKAZUZ4KB4TUFLQZ5ANCNFSM5FCR5PKQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by esbat-max** [reply]
Hello weslambert,

What I want is the following:
"Divert the log source into a differently named index (other than the default so--prefixed index name"

Br,
E

**Comment by esbat-max** [comment]
@dougburks @defensivedepth 

any update from your side

**Reply by dougburks** [reply]
> @dougburks @defensivedepth

@esbat-max Please do not tag an individual in a discussion unless they have already volunteered to help you in that discussion.

**Reply by esbat-max** [reply]
Hello Dough,

I apologize for this, I did not know the rules of the discussions.

Thank you,
E
